catt/controllers.py

```diff
@@ -54,15 +86,23 @@ def clear(self):
 
 
 class CastController:
-    def __init__(self):
+    def __init__(self, device_name):
         cache = Cache()
+        cached_ip = cache.get(device_name)
```

Comment:
Yes, but the problem is that, in the case of a cache miss, we are doing `get_chromecast(friendly_name=device_name)` in `CastController.init`. That would not be possible without the full name. Meaning that we then instead would need to do `get_chromecasts()` and then try and determine what cc the user ment. Not worth it if you ask me. Plus all the added complexity of needing to handle the possibility of multiple hits, we cant rely on the user making nice queries.